client/sites/components/sites-dashboard.tsx

```diff
@@ -43,6 +44,7 @@ import {
 	CALYPSO_ONBOARDING_TOURS_EVENT_NAMES,
 	useOnboardingTours,
 } from '../onboarding-tours';
+import DashboardBackportSitesList from '../v2/sites-list';
```

Comment:
It's unfortunate that this JS gets loaded regardless of the feature flag state. That's 100KB being added to many of the bundles.

I know you listed performance out of scope for this PR, but once merged it will start shipping a lot of dead code to all users. I assume a simple fix is to load this component async? I don't know if async loading is the right thing to do long term, but it just seems unfortunate to ship all that extra JS to existing users.

Reply:
Done by [fff1481](https://github.com/Automattic/wp-calypso/pull/104414/commits/fff1481dc8b961e6878d27f7294bb60879a88b7d).

Reply:
Great! [The bundle size changes](https://github.com/Automattic/wp-calypso/pull/104414#issuecomment-2999332470) look much better.